src/ukify/ukify.py

```diff
@@ -345,6 +360,17 @@ def check_inputs(opts):
     check_splash(opts.splash)
 
 
+def check_cert_and_keys_nonexistent(opts):
+    # Raise if any of the keys and certs are found on disk
+    paths = itertools.chain(
+        (opts.sb_key, opts.sb_cert),
+        *((priv_key, pub_key)
+          for priv_key, pub_key, _ in key_path_groups(opts)))
+    for path in paths:
+        if path and path.exists():
+            raise ValueError(f'{path} is present')
```

Comment:
```suggestion
    if any(path and path.exists() for path in paths):
        raise ValueError(f'{path} is present')
```

Reply:
No, that doesn't work, because `path` is not defined. I think it's important to include the path in the error message.